src/node.h

```diff
@@ -473,6 +473,17 @@ NODE_EXTERN v8::MaybeLocal<v8::Value> LoadEnvironment(
     std::unique_ptr<InspectorParentHandle> inspector_parent_handle = {});
 NODE_EXTERN void FreeEnvironment(Environment* env);
 
+// Set a callback that is called when process.exit() is called from JS.
```

Comment:
`called when process.exit() is called from JS.` might make some users think that the callback in invoked after the default `process.exit` has been actioned?

`called in response to process.exit() call from JS.` may be a better wording?

Reply:
> `called when process.exit() is called from JS.` might make some users think that the callback in invoked after the default `process.exit` has been actioned?

The default `process.exit()` terminates the process, which by definition precludes any other code from being run afterwards, so I don’t think that that is a very likely reading of this phrasing (or at least one that would appear self-contradictory).

> `called in response to process.exit() call from JS.` may be a better wording?

I’d be worried that that makes it sound like an asynchronous call, when it is really called during `process.exit()` itself.

I’ve added wording that explicitly specifies that this overrides the default handler, I hope that helps.